ops/ccs-ops-misc/del-old-snapshots/delete-old-snapshots.py

```diff
@@ -0,0 +1,49 @@
+import boto3
+import argparse
+from datetime import datetime, timezone
+
+# filter on snapshot description
+FILTER_STRING = 'Created*policy-0abe3036d255e9ae0*Daily*'
+
+# older than 21 days
+NUM_DAYS = 21
+
+
+def main():
+    # parse arguments
+    parser = argparse.ArgumentParser(
+        description='Batch delete daily snapshots older than 21 days')
```

Comment:
```suggestion
        description='Batch delete daily snapshots older than {days} days'.format(days=NUM_DAYS))
```

**Totally optional nitpick**: if we're setting the number of days in a global constant, we might want to use the constant here instead of hard-coding it, just to keep it consistent if we change the value at some point in the future.